src/implementations/twig/components/picture/picture.html.twig

```diff
@@ -0,0 +1,47 @@
+{% apply spaceless %}
+
+{#
+  Parameters:
+    - "picture" (associative array) (default: {}): 
+      - "img" (associative array) (default: {}):
+        - "src" (string) (default: ''): Path to the image
+        - "alt" (string) (default: ''): Alt text of the image
+      - "sources" (array) (default: []): format: [
+        {
+          "src" (string) (default: ''): Path to the source image
+          "media" (string) (default: ''): Media condition to use this source. Could be a breakpoint ('s', 'm', 'l', 'xl') or a free string.
+          "type" (string) (default: ''): Type of this source
+        },
+        ...
+      ]
+    - "picture_class" (optional) (string) (default: '')
+    - "image_class" (optional) (string) (default: '')
```

Comment:
our templates have extra_classes and extra_attributes by "default" forthe root element, why aren't we using them here..?